Examples/SwiftExample/SwiftExample/AppDelegate.swift

```diff
@@ -1,5 +1,6 @@
 import UIKit
 import VelocidiSDK
+import AppTrackingTransparency
```

Comment:
Can we comment somewhere that the example projects must be compiled using Xcode 12?

Reply:
I put that in the example's `README`: 712566f. 
Is this enough?